src/main/resources/resource/Arduino/generate/arduinoMsgs.schema

```diff
@@ -71,9 +71,14 @@
 < publishI2cData/deviceId/[] data
 
 # NeoPixel
-> neoPixelAttach/deviceId/pin/b32 numPixels/depth
-> neoPixelSetAnimation/deviceId/animation/red/green/blue/b16 speed
+> neoPixelAttach/deviceId/pin/b16 numPixels/depth
+> neoPixelSetAnimation/deviceId/animation/red/green/blue/white/b32 wait_ms
```

Comment:
why did speed change from 16 to 32 bit here (now wait_ms) ?

Reply:
because it "is" wait_ms per animation frame
which is far less confusing than the previously defined speed which was based on "who knows what" relativistic attribute (not fps) - therefore it will be in the range of ms to wait for a frame which potentially might be seconds or larger